CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
+
+[Full Changelog](https://github.com/robotika/osgar/compare/v0.0.1...master)
+
+
+## [v0.1.0](https://github.com/robotika/osgar/tree/v0.1.0) (2018-11-DD)
+[Full Changelog](https://github.com/robotika/osgar/compare/v0.0.1...v0.1.0)
+
+**osgar:**
+- Rename `robot.py` to `record.py` to better match record/replay pair
+- Extend `bus` to log errors, sleep and return timestamp also for `publish()`
+- Assert long queues (i.e. slow processing)
```

Comment:
This is only during replay, right?

Can we reword it a bit? We don't "assert long queues", right? We assert that when publishing, the oldest input message in the queue is not older than 0.1s - if I read 6e77b3bf3d3f29 right.

Reply:
OK, is it better now?

Reply:
Well, yes. One nit - I think we are missing "not" in the sentence. We assert the response time is **not** over 0.1 s and the queue is **not** long, right?

Reply:
you are right, thanks, fixed